securedrop/store.py

```diff
@@ -13,16 +13,29 @@
 from werkzeug.utils import secure_filename
 
 from secure_tempfile import SecureTemporaryFile
+
 from worker import rq_worker_queue
 
 
+import typing
+
+if typing.TYPE_CHECKING:
+    # flake8 can not understand type annotation yet.
+    # That is why all type annotation relative import
+    # statements has to be marked as noqa.
+    # http://flake8.pycqa.org/en/latest/user/error-codes.html?highlight=f401
+    from typing import Optional, Union, List, Any  # noqa: F401
+    from tempfile import _TemporaryFileWrapper  # noqa: F401
+    from io import BufferedIOBase  # noqa: F401
+    from sqlalchemy import model  # noqa: F401
+    from flask_sqlalchemy import Submission, Reply, Session  # noqa: F401
```

Comment:
I think we want:
```
from sqlalchemy import Session
from models import Reply, Submission
```
It's hard to catch these things right now, since mypy isn't complaining about these usages (see the comment below about sqlalchemy models effectively not being type checked).